data/core/rootview.lua

```diff
@@ -293,12 +293,12 @@ function RootView:on_mouse_moved(x, y, dx, dy)
   end
 
   local div = self.root_node:get_divider_overlapping_point(x, y)
-  local tab_index = self.overlapping_node and self.overlapping_node:get_tab_overlapping_point(x, y)
+  local tab_height = self.overlapping_node and self.overlapping_node:should_show_tabs() and select(4, self.overlapping_node:get_scroll_button_rect(1))
```

Comment:
This doesn't work on tab bars that are not at the top.

Reply:
Fixed.